src/v8/mui-x-upgrade.ts

```diff
@@ -0,0 +1,25 @@
+import { existsSync } from "fs";
+
+import { executeCommand } from "../util/execute-command.util";
+import { updateDependencyVersion } from "../util/update-dependency-version.util";
+
+const adminPackageJsonPath = "admin/package.json";
+
+/**
+ * Update MUI X version to 6.0.0
+ *
+ * Migration Guide 5-6: https://mui.com/x/migration/migration-data-grid-v5
+ *
+ */
+export default async function updateMuiXVersion() {
+    if (!existsSync(adminPackageJsonPath)) {
+        return;
+    }
+
+    await updateDependencyVersion(adminPackageJsonPath, "@mui/x-data-grid", "^6.0.0");
+    await updateDependencyVersion(adminPackageJsonPath, "@mui/x-data-grid-pro", "^6.0.0");
```

Comment:
This will likely fail due to incorrect peer dependencies when upgrading the Comet packages to v8. We should add a separate script in the before-install stage that changes the version numbers in the package.json.